### ISSUE TITLE
	 Not able to invoke the checkboxes in 'Fields controls mappings pluggin' dialog in High DPI/Zoom/Reflow mode.

### ISSUE THREAD

**Issue by msftedad** [tinymce/tinymce#8913]
**What is the current behavior? Describe the bug**
Not able to invoke the checkboxes in 'Fields controls mappings pluggin' dialogue in High DPI mode. upon clicking on check boxes dialog moves to upward but check box is not getting selected/deselected.
**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**
Prerequisites:

1. For High DPI, Settings > Display > scale > 125%
2. For Zoom and Reflow, Go to Settings->System->Display->Resolution->1280*720.
3. Set Browser zoom to 200% and 400%.

Repro steps:
1. Navigate to any Engagement after opening the link and invoke it.
2. Now invoke 'Edit Engagement' button.
3. Now invoke 'Feedback' tab.
4. Now navigate to any option in left side like 'Scenarios' and invoke it. Sub control would appear.
5. Invoke 'Form Template'.
6. Now navigate to 'Template' and invoke it.
7. Wait for template to be load.
8. Now navigate to 'Fields' in toolbar and invoke it.
9. 'Fields controls mappings pluggin' dialog will appear.
10. Try to invoke the checkboxes present in the dialogue.
11. Observe the issue in High DPI mode. 

**What is the expected behavior?**
User should be able to invoke the checkboxes in 'Fields controls mappings pluggin' dialogue in High DPI/Zoom/Reflow mode as it is invoking in normal mode. 
**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
OS Version: Windows 11 Dev 22H2(OS Build 25276.1000)
Edge Dev Browser Version: Version 111.0.1660.2 (Official build) dev (64-bit)
URL: [Engagements - Microsoft Collaborate (microsoft-tst.com)](https://partner.microsoft-tst.com/en-us/dashboard/collaborate/engagements)

Attachments:
[Solid square screen reader attachment.webm](https://github.com/tinymce/tinymce/assets/93735775/0a7667b4-f820-45e5-81e0-1daf77a7f16c)
[High dpi.webm](https://github.com/tinymce/tinymce/assets/93735775/db24251e-a5e7-489e-bbe3-086378a97973)

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi team, Any update on this issue?
